tests/helics/application_api/FederateTests.cpp

```diff
@@ -810,7 +810,7 @@ TEST(federate_tests, forceErrorPendingTimeAsync)
 
     Fed1->enterExecutingMode();
     Fed1->requestTimeAsync(2.0);
-    Fed1->error(9827);
+    Fed1->globalError(9827);
```

Comment:
Was making this test use `globalError` intentional?

Reply:
sort of.  It doesn't impact the test results and I wanted the function used for code coverage purposes.  This was an easy way to do that.    local or Global could be used interchangeably here.